mne/utils/docs.py

```diff
@@ -25,9 +25,13 @@
 # Verbose
 docdict['verbose'] = """
 verbose : bool | str | int | None
-    If not None, override default verbose level (see :func:`mne.verbose`
-    and :ref:`Logging documentation <python:tut-logging>` for more).
-    If used, it should be passed as a keyword-argument only."""
+    Controls the verbosity of the generated logging output. If ``True``, sets
+    the log level to ``INFO``, and if ``False``, sets it to ``WARNING``. If
+    ``None``, use the default logging level. Accepts all
+    `Python logging levels <https://docs.python.org/3/library/logging.html#logging-levels>`_
+    (case ignored).
+
+    Should only be passed as a keyword argument."""  # noqa: E501
```

Comment:
```suggestion
    Controls the verbosity of logging output. Accepts all
    `Python logging levels <https://docs.python.org/3/library/logging.html#logging-levels>`_ (case ignored). For most use cases, setting either
    ``True`` (equivalent to "info") or ``False`` (equivalent to "warning") will
    suffice. ``None`` selects the default logging level. Can only be passed as
    a keyword (not positional) argument."""  # noqa: E501
```

suggested edit to be a little more succinct (?)

Reply:
oh, never mind this suggestion, I just saw @larsoner's point about referring to our logging docs instead of the python docs.